app/app.go

```diff
@@ -847,19 +900,6 @@ func NewWasmApp(
 	// upgrade.
 	app.setPostHandler()
 
-	// At startup, after all modules have been registered, check that all proto
-	// annotations are correct.
-	protoFiles, err := proto.MergedRegistry()
-	if err != nil {
-		panic(err)
-	}
-	err = msgservice.ValidateProtoAnnotations(protoFiles)
-	if err != nil {
-		// Once we switch to using protoreflect-based antehandlers, we might
-		// want to panic here instead of logging a warning.
-		_, _ = fmt.Fprintln(os.Stderr, err.Error())
-	}
-
```

Comment:
Just curious: Why is this removed?

Reply:
Because it was removed from `simapp/app.go` and moved to `baseapp/baseapp.go` in cosmos-sdk . See https://github.com/cosmos/cosmos-sdk/pull/24450